package-lock.json

```
@@ -47,6 +48,9 @@
         "type-fest": "4.30.1",
         "typescript": "^5.7.2",
         "vite": "^5.4.11"
+      },
+      "engines": {
+        "node": ">=22.0.0"
```

This was updated when running npm install locally:

```suggestion
        "node": ">=20.0.0"
```

---

Oops, missed that, thanks!